book/security.rst

```diff
@@ -869,7 +871,8 @@ Checking to see if a User is Logged In (IS_AUTHENTICATED_FULLY)
 
 So far, you've checked access based on roles - those strings that start with
 ``ROLE_`` and are assigned to users. But if you *only* want to check if a
-user is logged in (you don't care about roles), then you can see ``IS_AUTHENTICATED_FULLY``::
+user is logged in (you don't care about roles), then you can see
```

Comment:
Instead of `you can see IS_AUTHENTICATED_FULLY::` I think it should be `you can use IS_AUTHENTICATED_FULLY::`